closure/goog/net/xhriopool.js

```diff
@@ -65,6 +78,9 @@ goog.net.XhrIoPool.prototype.createObject = function() {
       xhrIo.headers.set(key, value);
     });
   }
+  if (this.withCredentials_) {
+    xhrIo.setWithCredentials(true);
```

Comment:
Can you add a unit test that verifies withCredentials is set for XhrIos created using XhrIoPool?